src/App.jsx

```diff
@@ -14,57 +15,82 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [message, setValue] = useState('Jam is selected');
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [isButtonVisible, setButtonVisible] = useState(true);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleAddClick = good => {
+    setSelectedGood(good);
+    setValue(`${good} is selected`);
+    setButtonVisible(true);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const handleRemoveClick = () => {
+    setSelectedGood(null);
+    setValue('No goods selected');
+    setButtonVisible(false);
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  const handleClearClick = () => {
+    setSelectedGood(null);
+    setValue('No goods selected');
+    setButtonVisible(false);
+  };
```

Comment:
The `handleRemoveClick` and `handleClearClick` functions perform the same actions. Consider refactoring by creating a single function to handle both cases, which will reduce redundancy and improve maintainability.